contribution-guidelines.md

```diff
@@ -19,17 +19,19 @@ Thank you for your interest in contributing to the Tech Start UCalgary website r
 ## 👩‍⚖️ Admins
 
 | Name                 | GitHub Profile                              |
-|----------------------|---------------------------------------------|
+| -------------------- | ------------------------------------------- |
 | Joel Happ            | [Jhappy77](https://github.com/Jhappy77)     |
 | Niyousha Raeesinejad | [Niyousha99](https://github.com/Niyousha99) |
-| Ben Schmidt          | [Anidion](https://github.com/Anidion)     |
+| Ben Schmidt          | [Anidion](https://github.com/Anidion)       |
```

Comment:
Can you add yourslef and @abodthedude25 to the admins table as well? Thanks!

Reply:
✔️